fe/fe-core/src/main/java/com/starrocks/common/util/NetUtils.java

```diff
@@ -67,9 +70,25 @@ public static List<InetAddress> getHosts() {
             Enumeration<InetAddress> a = e.getInetAddresses();
             while (a.hasMoreElements()) {
                 InetAddress addr = a.nextElement();
+                // IPv6 address starting with fe80 (Link-local Address) is not supported for now.
+                if (addr instanceof Inet6Address &&
```

Comment:
169.254.0.0/16 is a link-local ipv4 address, do we need to filter it out together?

Reply:
didn't see the need for now

Reply:
what's the necessity to treat them differently?

Reply:
not sure about the reason, for now, i only found the issue with ipv6 link local addr, ipv4 is fine

Reply:
so think it this way, what's the downside if filtering out ipv4 link-local address as well? our application will be more robust if the user's env happens to have a link-local ipv4 address bind to its interface, right?

And from the code, the `instanceof` check is no longer needed.